docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,262 @@
+# Installing Canonical Kubernetes Offline or in an air gapped environment
+
+There are situations where it is necessary or desirable to run Canonical 
+Kubernetes on a machine that is not connected to the internet. 
+Based on different degrees of separation from the network,
+different solutions are offered to accomplish this goal.
+This guide explains the necessary preparation required for the
+offline installation and walks you through the different potential scenarios.
```

Comment:
This text should be useful for people that look to install on airgap environments, and is not every effective at that as written. Also, we should avoid _talking directly_ to the person. 

Perhaps something like this would work better:

> This guide explains the necessary extra preparation that is needed for airgap deployments, as well the steps that are needed to successfully deploy Canonical Kubernetes in such environments.